distribution/lib/Standard/Table/0.0.0-dev/src/Data/Conversions/Convertible_To_Columns.enso

```diff
@@ -15,6 +16,10 @@ type Convertible_To_Columns
 Convertible_To_Columns.from (that:JS_Object) =
     Convertible_To_Columns.Value that.field_names that.get
 
+## PRIVATE
+Convertible_To_Columns.from (that:Jackson_Object) =
+    Convertible_To_Columns.Value that.field_names that.get
```

Comment:
I'm a bit concerned that we now start maintaining two separate JSON hierarchies and this will require us to define duplicate methods for both, like here.

For us it's just some additional maintenance - we can live with it.

But I'm worried that users may end up getting problems if they define an operation that works on just one case and then will be surprised it doesn't work when they e.g. switch to Jackson. Though, admittedly, at least currently I expect that to be rare.

Shall one replace the other? Shall we use some polyglot tricks to make them interchangeable (not exactly if it's possible)? Not necessary to answer these right now, but just noting these questions, so that we keep in mind we may need to address them somehow later on.

Reply:
I think I can make it so the Jackson parser produces a JS Object as well. The walker needs to be clever but it should work.

Not one I want to do in this PR but will add as a follow up.

Reply:
Sure, for this PR this is all fine, I just wanted us to not forget to address this at some point. I'm worried if producing JS Object won't take away the efficiency we are gaining by Jackson? Well, we can try and see :)

Reply:
Added #8914 to follow this up.